offline/docs_ubuntu_22.04.md

```diff
@@ -1,80 +1,24 @@
-# How to install wire
+# How to install wire (offline cluster)
 
 We have a pipeline in  `wire-server-deploy` producing container images, static
 binaries, ansible playbooks, debian package sources and everything required to
 install Wire.
 
-## Installing docker
+## Preparations
 
-Note: If you are using a Hetzner machine, docker should already be installed (you can check with `docker version`) and you can skip this section.
+This section is a continuation of the demo cluster setup described in single_hetzner_machine_installation.md. At this point, the following prerequisites should be met:
```

Comment:
This document is certanly not a continuation: THIS IS WHERE CUSTOMERS START.

Reply:
EG: in a customer deployment, we point them to this document, and follow along with them, or they follow along with us.